x-pack/plugins/cases/public/components/app/use_available_owners.ts

```diff
@@ -16,16 +19,33 @@ import { useKibana } from '../../common/lib/kibana';
  * both solutions use `crud_cases` and `read_cases` capability names
  **/
 
-export const useAvailableCasesOwners = (level: 'crud' | 'read' = 'crud'): string[] => {
-  const { capabilities } = useKibana().services.application;
-  const capabilityName = `${level}_cases`;
-  return Object.entries(capabilities).reduce(
-    (availableOwners: string[], [featureId, capability]) => {
-      if (featureId.endsWith('Cases') && !!capability[capabilityName]) {
-        availableOwners.push(featureId.replace('Cases', ''));
+export const useAvailableCasesOwners = (
+  capabilities: Capability[] = ['create', 'read', 'update', 'delete', 'push']
+): string[] => {
+  const { capabilities: kibanaCapabilities } = useKibana().services.application;
+
+  return Object.entries(kibanaCapabilities).reduce(
```

Comment:
I thought it made more sense to specify the exact permissions instead of all or read, so we need a loop to check if a feature has each of the capabilities specified (this defaults to all of them).